t/t2219-job-manager-restart.t

```diff
@@ -78,6 +88,22 @@ test_expect_success 'verify that named queue status persists across restart' '
 	grep "^batch: Job submission is disabled: xyzzy" dump_queue_dis.out
 '
 
+test_expect_success 'verify that named queue stopped persists across restart' '
+	mkdir -p conf.d &&
+	cat >conf.d/queues.toml <<-EOT &&
+	[queues.debug]
+	[queues.batch]
+	EOT
+	flux start -o,--config-path=$(pwd)/conf.d \
+	    -o,-Scontent.dump=dump_queue_stopped.tar \
+	    flux queue stop --queue batch xyzzy &&
+	flux start -o,--config-path=$(pwd)/conf.d \
+	    -o,-Scontent.restore=dump_queue_stopped.tar \
+	    flux queue status >dump_queue_stopped.out &&
+	grep "^debug: Scheduling is started" dump_queue_stopped.out &&
+	grep "^batch: Scheduling is stopped: xyzzy" dump_queue_stopped.out
+'
+
```

Comment:
do we also need targeted tests for
- configured queue is missing from checkpoint (enable/stop should get default values)
- checkpointed queue is missing from config (it should be ignored)